src/Repository/MissionRepository.php

```diff
@@ -43,7 +43,10 @@ public function findByOrganizationQb(Organization $organization): QueryBuilder
 
     public function findByFilters(array $filters): array
     {
-        return $this->findByFiltersQb($filters)->getQuery()->getResult();
+        $qb = $this->findByFiltersQb($filters);
+        $qb->setMaxResults(100); // TODO Paginate
```

Comment:
We should remove this security in this PR, can you check every call to this function ?

Reply:
You mean the ```setMaxResults(100)``` ?
The ```findByFilters``` method is used in 
- MissionController -> fullList()
- AddToMissionModalController
I changed it for pagination mode in fullList MissionController but for AddToMisisonModalController it is not as easy as it loaded in modal with ajax call. I can have a look to do it with ajax but it won't be so fast ^^

Reply:
ok, then can you remove the TODO ?